libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -762,6 +762,12 @@ namespace SohImGui {
                 EnhancementCheckbox("60FPS Interpolation", "g60FPS");
                 EnhancementCheckbox("Disable LOD", "gDisableLOD");
                 Tooltip("Turns off the level of detail setting, making models always use their higher poly variants");
+                EnhancementCheckbox("Disable Draw Distance", "gDisableDrawDistance");
+                Tooltip("Turns off the objects draw distance, making objects being visible from a longer range");
+                if (CVar_GetS32("gDisableDrawDistance", 0) == 1) {
+                    EnhancementCheckbox("Kokiri Draw Distance", "gDisableKokiriDrawDistance");
+                    Tooltip("Kokiris are mystical being that appear from a certain distance\nEnable this will remove their draw distance\nNeeds to reload the map to take effect");
```

Comment:
Language tweak:  "Kokiri are mystical beings that only appear from a certain distance away
Enabling this will remove their draw distance limitation
Requires reloading the map to take effect"